addons/shaker/shaker.gd

```diff
@@ -0,0 +1,108 @@
+class_name Shaker
+extends Node
```

Comment:
All the plugin provides is this `addons/shaker/shaker.gd` script. I think that we should be careful with the dependencies we add and weight the pros and cons. I can't forget that the well-stablished Dialogue Manager bited us today :) . A shake effect can be done with a simple AnimationPlayer or Tween. So I would add our custom implementation rather than adding a dependency for this case.

Reply:
Yeah, you make a good point 🤔 . I think I got a bad habit of adding that plugin to most projects without thinking too much about it 😅 .
In this case, I think I'd just remove the shaking effect, iirc it was used for a character animation and for a camera shake, and none of those things are just details and aren't really necessary for this PR.

I think it would be a good idea to have a bit of camera shake for juiciness, but that can be done later in a separate PR and we can implement it ourselves. So, I think I'll just remove the plugin and its uses for this PR. What do you think? @manuq

Reply:
@JuanFdS yes, removing the effect for now sounds good!